app/src/main/res/values/strings.xml

```diff
@@ -115,4 +115,10 @@
     <string name="title_no_salon">No Salon</string>
     <string name="action_change">Change</string>
     <string name="action_change_status">Change Status</string>
+    <string name="title_pick_image">pick image</string>
+    <string name="msg_request_read_camera">Please grant permission open camera permission  to
+        make an image</string>
+    <string name="action_agree">AGREE</string>
+    <string name="action_dis_agree">DIS AGREE</string>
+    <string name="msg_request_write_file">Write list device using</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.msg_request_write_file` appears to be unused
 Unused resources make applications larger and slow down builds.